quarkus/distribution/scripts/run.sh

```diff
@@ -16,24 +16,30 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-#
-
-# Linux Quarkus fast-jar run script for Apache Polaris
 
-set -e
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+cd "${SCRIPT_DIR}"
 
-script_dir="$(dirname "$0")"
+# Default to server if no component specified
+COMPONENT=${1:-server}
```

Comment:
I'm not sure the default is useful. It is possible only if `server` is executed without arguments. :thinking:

Reply:
Does `server` take any parameters? Even if yes, I think the no-arg run is still the majority, in that case, it's easier to use by just hit `run.sh`.

Reply:
why not have `bin/polaris-server` and `bin/polaris-admin` (note: no `.sh` suffix) similar to Spark's `bin/spark-shell`, `bin/spark-sql`, etc?..

Reply:
I think it's a good idea to have two different bash script, but I'd prefer to do that once we merge libs between admin tool and server(the third todo item)

Reply:
This PR refactors run scripts to support a combined dist archive anyway. I think it makes sense to put separate admin/server scripts in this PR.

Reply:
Fixed in the new commit.